README.md

```diff
@@ -22,11 +22,11 @@ The following are not in scope for this project:
 
 Hardening applied:
 
-- Setting numerous hardened sysctl values (Inspired by but not the same as Kicksecure's)
+- Setting numerous hardened sysctl values (Inspired by but not the same as Kicksecure's) <sup>[details](https://github.com/secureblue/secureblue/tree/live/config/files/usr/etc/sysctl.d)</sup>
```

Comment:
Looks good, very useful! One change though is can you point this link directly at the file? Thanks

Reply:
Resolved https://github.com/secureblue/secureblue/pull/191/commits/e9dd15dd8208f4da5a8b0136780ea8ee4d289349